### PR TITLE
IP-2226: Critical edits for iEMA PR

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: id/iema/2016/index.html [hunk 1]
```diff
@@ -105,55 +105,55 @@
 
                 <div class="description">
                   <h3>Untuk Segera Dipublikasikan</h3>
-                  Indonesia, 8 Maret 2017 Indonesia, 8 Maret 2017 – Beberapa startup Indonesia Bukalapak, Mazaya, Blibli, Matahari
-Mall,Muslim Market dan juga 23 E-commerce Asia Tenggara lainnya berkesempatan untuk
-mendatangi perhelatan akbar tahunan <strong>iPrice E-Commerce Merchant Awards</strong> 2016 pada
-konferensi eTail Asia di Marina Bay Sands, Singapura. Penghargaan ini diselengkarakan oleh
-iPrice Group yang berkolaborasi dengan eTail Asia, Trusted Company & UseInsider.<br><br>
+                  Indonesia, 8 Maret 2017 – Beberapa startup Indonesia Bukalapak, Mazaya, Blibli, Matahari
+                  Mall,Muslim Market dan juga 23 E-commerce Asia Tenggara lainnya berkesempatan untuk
```
Comment (abidulrmdn) [inline id/iema/2016/index.html]: space after ','

File: id/iema/2016/index.html [hunk 2]
```diff
@@ -105,55 +105,55 @@
...
+              </div>
+              <div class="caption_container">  
+                <div class="caption">
                     David Chmelař, Co-Founder dan CEO dari iPrice Group memperkenalkan iEMA Awards di Konferensi eTail Asia</div>
                 </div>
                 <div class="description">
                     Dari berbagai kategori penghargaan yang ada, startup Indonesia Bukalapak berhasil mencuri
-perhatian dengan “memborong” pulang 2 penghargaan sekaligus, Popularity Award & juga
-The Absolute Winner dari iEMA 2016.
+                    perhatian dengan “memborong” pulang 2 penghargaan sekaligus, Popularity Award & juga
```
Comment (abidulrmdn) [inline id/iema/2016/index.html]: "Award  & juga"
dont use '&' character.. use the code that presents it.

File: id/iema/2016/index.html [hunk 3]
```diff
@@ -105,55 +105,55 @@
...
-dengan adanya layanan pengiriman cepat dalam sehari bekerjasama dengan perusahaan
-logistik terkemuka, layanan call center yang siap sedia melayani pelanggan selama 24 jam,
-kecepatan loading website dan aplikasi yang cepat, serta adanya beberapa fitur terbaru salah
-satunya adalah BukaReksa, yang mana merupakan fitur yang menyediakan sarana
-berinvestasi reksa dana.<br /><br />
+                    pendiri Bukalapak) memiliki pengalaman membuat dan memasarkan website di Suitmedia,
+                    sadar bahwa para seller dan buyer menginginkan sebuah situs online marketplace yang bisa
+                    mempertemukan antara penjual dan pembeli secara aman (tanpa penipuan). Mereka pun
+                    akhirnya membuat sebuah situs online marketplace bernamakan Bukalapak.com<br /><br />
+                    Kreatifitas & juga keberanian tim Bukalapak dalam membuat kampanye pemasaran yang
```
Comment (abidulrmdn) [inline id/iema/2016/index.html]: no "&"

File: id/iema/2016/index.html [hunk 4]
```diff
@@ -105,55 +105,55 @@
...
+                    “nyentrik” membuat startup Indonesia ini memiliki tempat tersendiri di hati para pelanggan.
+                    Disamping itu, semangat untuk memajukan UKM di Indonesia yang dimiliki Bukalapak juga
+                    menjadi nilai unik tersendiri yang tidak begitu ditekankan loleh kompetitor lain.<br /><br />
+                    Evi Andarini juga mengatakan bahwa, kedepannya Bukalapak akan terus berinovasi untuk
+                    memenuhi kebutuhan penjuan & juga pembeli. Penerapan inovasi tersebut ditujukkan
```
Comment (abidulrmdn) [inline id/iema/2016/index.html]: "&"

**Review by sandyandi-iprice — APPROVED** [review]
LGTM

File: id/iema/2016/index.html [hunk 5]
```diff
@@ -105,55 +105,55 @@
...
+                    dengan adanya layanan pengiriman cepat dalam sehari bekerjasama dengan perusahaan
+                    logistik terkemuka, layanan call center yang siap sedia melayani pelanggan selama 24 jam,
+                    kecepatan loading website dan aplikasi yang cepat, serta adanya beberapa fitur terbaru salah
+                    satunya adalah BukaReksa, yang mana merupakan fitur yang menyediakan sarana
+                    berinvestasi reksa dana.<br /><br />
```
Comment (abidulrmdn) [inline id/iema/2016/index.html:156]: why <br>?
use styling

**Review by abidulrmdn — CHANGES_REQUESTED** [review]

File: id/iema/2016/index.html (hunk 5 above)
Comment (ghost) [reply id/iema/2016/index.html:156]: using reindent align to make code clearly

File: id/iema/2016/index.html (hunk 1 above)
Comment (ghost) [reply id/iema/2016/index.html]: added

File: id/iema/2016/index.html (hunk 2 above)
Comment (ghost) [reply id/iema/2016/index.html]: replaced

File: id/iema/2016/index.html (hunk 3 above)
Comment (ghost) [reply id/iema/2016/index.html]: replaced

File: id/iema/2016/index.html (hunk 4 above)
Comment (ghost) [reply id/iema/2016/index.html]: replaced

**Review by abidulrmdn — APPROVED** [review]

**Comment by abidulrmdn** [comment]
LGTM

**Pull request merged**